### ISSUE TITLE
Doesn't work on latest Visual Studio

### ISSUE THREAD

**Issue by Renha** [Dealman/BannerlordModTemplate#7]
Trued installing on Visual Studio Community 2022, which has failed. If for modding specific version is required, is unknown for me.

**Comment by Dealman** [comment]
This thing is 3 years old by now, chances are VS has changed the way extensions are handled. Do you get any error messages?

**Comment by Arman-Espiar** [comment]
Yes, this plugin does not support visual studio 2022 and probably the latest version of C# language. Can you add visual studio 2022 support to it? Due to the lack of support for Visual Studio 2022, it does not even appear in the list of extensions.

**Comment by Dealman** [comment]
I'd recommend you try and use this instead;
https://github.com/BUTR/Bannerlord.Module.Template

I was working on an updated version for VS2022, but it feels a bit redundant if this works just as well with a few more options exposed.
